src/backend/services/workflow_service.py

```
@@ -767,7 +779,12 @@ def run_agentic_workflow(natural_language_query: str, model_provider: str, model
                 unified_metrics.llm_cleaning_stats = llm_monitor.get_numeric_stats()
 
             collector = get_workflow_metrics_collector()
-            collector.record_workflow(unified_metrics)
+            _run_org_id: str | None = None
+            try:
+                _, _run_org_id = get_run_registry().get_run_owner(workflow_id)
+            except Exception:
+                pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)